substrate-node/pallets/pallet-smart-contract/src/mock.rs

```diff
@@ -3,11 +3,12 @@ use std::{panic, thread};
 
 use super::*;
 use crate::name_contract::NameContractName;
-use crate::{self as pallet_smart_contract, types::BlockNumber};
+use crate::{self as pallet_smart_contract};
 use codec::{alloc::sync::Arc, Decode};
 use frame_support::{
     construct_runtime, parameter_types,
     traits::{ConstU32, GenesisBuild},
+    weights::PostDispatchInfo,
```

Comment:
Just a thought (nothing to do with this review): can we maybe start using the style of importing shown below? It would make rebasing branches much easier (no merge conflicts). 
```
use frame_support::construct_runtime;
use frame_support::parameter_types;
use frame_support::traits::ConstU32;
use frame_support::traits::GenesisBuild;
use frame_support::weights::PostDispatchInfo;
```

Reply:
Not really a fan of your suggestion, it's rust standard to group imports (it also looks cleaner imo)